### ISSUE TITLE
[TECH ED] Record a Goose

### ISSUE THREAD

**Issue by Mgphone** [Mgphone/My-Coursework-Planner#40]
### Link to the coursework

https://github.com/CodeYourFuture/Project-Record-A-Goose

### Why are we doing this?

A mission of discovery for you.

1. Go to https://github.com/CodeYourFuture/CYF-Record-A-Goose-Project
2. Using Devtools, Lighthouse, and the WAVE accessibility plugin, find all the ways this app is broken
3. How would you fix the problems? Discuss in **a thread** in your channel. 

This is also a fun in-person workshop, so you could choose to do it in a class or coworking space together.

https://record-a-goose.onrender.com/

### Maximum time in hours

2

### How to get help

You could choose to do this in class together during independent study time.

### How to submit

Write up your findings in a thread in your class channel. 

Don't make a new post each. Discuss in a thread.

### How to review

It is a review! ;)

### Anything else?

* https://developer.chrome.com/docs/lighthouse/overview/
* https://wave.webaim.org/extension/ 
* https://accessibilityinsights.io/docs/web/overview/ 
* https://design-system.service.gov.uk/

**Comment by Mgphone** [comment]
1.In the Do You like Geese ther eis two errors
**a.**Element has insufficient color contrast of 2.7 (foreground color: #9d9da1, background color: #ffffff, font size: 14.3pt (19px), font weight: normal). Expected contrast ratio of 4.5:1Element has insufficient color contrast of 2.7 (foreground color: #9d9da1, background color: #ffffff, font size: 14.3pt (19px), font weight: normal). Expected contrast ratio of 4.5:1
Use foreground color: #717173 and the original background color: #ffffff to meet a contrast ratio of 4.87:1.
**b.**Document does not have a non-empty <title> element

2.In the Tab of What type of goosed did you see have one errors And
In the Tab of When did you see the goose have same errors
**a.**ensures iframes and element have an accessible name

_Fix ONE of the following:Element has no title attributearia-label attribute does not exist or is emptyaria-labelledby attribute does not exist, references elements that do not exist or references elements that are emptyElement's default semantics were not overridden with role="none" or role="presentation"_
--

3.Goose recorded - Record a goose sighting got one error
**a** In the image threre is no altrnative text
